test/WebJobs.Script.Tests.Perf.Dashboard/AppVeyorClient.cs

```diff
@@ -0,0 +1,231 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace WebJobs.Script.Tests.Perf.Dashboard
```

Comment:
Don't we have this in the E2E project? If so, let's reuse and avoid duplication.

Reply:
No, we do not have this code in the E2E project, this code is in the function app for E2E.